ext/bcmath/libbcmath/src/recmul.c

```diff
@@ -66,22 +64,21 @@ static bc_num new_sub_num(size_t length, size_t scale, char *value)
 static void _bc_simp_mul(bc_num n1, size_t n1len, bc_num n2, int n2len, bc_num *prod)
 {
 	char *n1ptr, *n2ptr, *pvptr;
-	char *n1end, *n2end;		/* To the end of n1 and n2. */
-	int indx, sum, prodlen;
+	char *n1end, *n2end;        /* To the end of n1 and n2. */
+	int sum = 0;
 
-	prodlen = n1len+n2len+1;
+	int prodlen = n1len + n2len + 1;
```

Comment:
Note to myself: why is ``n2len`` an int and probably should be ``size_t``. Ditto for ``prodlen``